### PR TITLE
Add last slide on first presentation

### PR DESCRIPTION
**Pull request by evrardjp**
Great commit msg btw.

-----
[View rendered 2019-01/PITCHME.md](https://github.com/evrardjp/presentations/blob/add_last_slide/2019-01/PITCHME.md)

### CODE REVIEW

**Comment by evrardjp** [comment]
https://gitpitch.com/evrardjp/presentations/add_last_slide?q=2019-01

File: 2019-01/PITCHME.md [hunk 1]
```diff
@@ -176,5 +175,12 @@ Note:
 Still for the context, other projects we are using behind the scenes.
 
 ---
-@title[What's in it for SUSE?]
+@title[What's the SUSE state in the community]
 
+#### State of OpenSUSE in the OpenStack community
```
Comment (dirkmueller) [inline 2019-01/PITCHME.md]: openSUSE

File: 2019-01/PITCHME.md [hunk 2]
```diff
@@ -176,5 +175,12 @@ Note:
...
+
+@ul[](false)
+- Currently, openSUSE is not listed in [popular distros](https://governance.openstack.org/tc/reference/project-testing-interface.html#pti-linux-distros).
+- openSUSE is used not only crowbar and Ardana, it's used in other places in the community.
+- Please help projects (give support in the OpenSUSE image builds in infra/dib, in rpm-packaging).
```
Comment (dirkmueller) [inline 2019-01/PITCHME.md]: openSUSE

File: 2019-01/PITCHME.md [hunk 3]
```diff
@@ -176,5 +175,12 @@ Note:
...
 
 ---
-@title[What's in it for SUSE?]
+@title[What's the SUSE state in the community]
 
+#### State of OpenSUSE in the OpenStack community
+
+@ul[](false)
+- Currently, openSUSE is not listed in [popular distros](https://governance.openstack.org/tc/reference/project-testing-interface.html#pti-linux-distros).
+- openSUSE is used not only crowbar and Ardana, it's used in other places in the community.
```
Comment (dirkmueller) [inline 2019-01/PITCHME.md]: Crowbar

File: 2019-01/PITCHME.md [hunk 4]
```diff
@@ -176,5 +175,12 @@ Note:
 Still for the context, other projects we are using behind the scenes.
 
 ---
-@title[What's in it for SUSE?]
+@title[What's the SUSE state in the community]
 
+#### State of OpenSUSE in the OpenStack community
+
+@ul[](false)
```
Comment (dirkmueller) [inline 2019-01/PITCHME.md:186]: Start with something positive, like:

```OpenStack tests against openSUSE Leap using our packaging```

and

```When it breaks, they ping dirk on irc```

and

```Occassionally dirk is not available```

File: 2019-01/PITCHME.md (hunk 3 above)
Comment (dirkmueller) [inline 2019-01/PITCHME.md]: I think the message would be:

```We need to support the community and be fixing issues before they arise```

**Comment by evrardjp** [comment]
Please refresh.

File: 2019-01/PITCHME.md [hunk 5]
```diff
@@ -176,5 +175,26 @@ Note:
...
 
 ---
-@title[What's in it for SUSE?]
+@title[What's the SUSE state in the community]
 
+#### State of openSUSE in the OpenStack community
+
+@ul[](false)
+- OpenStack tests against openSUSE Leap using our packaging
+- When it breaks, they ping dirk on irc
```
Comment (dirkmueller) [inline 2019-01/PITCHME.md]: they sometimes ping..

File: 2019-01/PITCHME.md [hunk 6]
```diff
@@ -176,5 +175,26 @@ Note:
...
+- Occassionally dirk is not available
```
Comment (dirkmueller) [inline 2019-01/PITCHME.md]: indent once more, and then "sometimes dirk.."

File: 2019-01/PITCHME.md [hunk 7]
```diff
@@ -176,5 +175,26 @@ Note:
...
+- We need to support the community and be fixing issues before they arise
```
Comment (dirkmueller) [inline 2019-01/PITCHME.md]: We need a better, more sustainable support for the community and take care of issues before they arise

File: 2019-01/PITCHME.md [hunk 8]
```diff
@@ -176,5 +175,26 @@ Note:
 Still for the context, other projects we are using behind the scenes.
 
 ---
-@title[What's in it for SUSE?]
+@title[What's the SUSE state in the community]
```
Comment (dirkmueller) [inline 2019-01/PITCHME.md]: What about SUSE in the community?

**Review by dirkmueller — APPROVED** [review]
some nits inside

File: 2019-01/PITCHME.md [hunk 9]
```diff
@@ -176,5 +175,26 @@ Note:
...
 
+#### State of openSUSE in the OpenStack community
+
+@ul[](false)
+- OpenStack tests against openSUSE Leap using our packaging
+- When it breaks, they ping dirk on irc
+- Occassionally dirk is not available
+- We need to support the community and be fixing issues before they arise
+- Please help projects (give support in the openSUSE image builds in infra/dib, in rpm-packaging).
```
Comment (cmurphy) [inline 2019-01/PITCHME.md]: Maybe split this in two:

```
Help support OpenSUSE in OpenStack by:
* Helping provide stable support for OpenSUSE images by helping maintain SUSE support for diskimage-builder, supporting the mirrors for the OpenDev (nee OpenStack) Infra team
* Help support community packages by contributing to the rpm-packaging project and keeping the Cloud:OpenStack:* OBS projects stable

File: 2019-01/PITCHME.md [hunk 10]
```diff
@@ -176,5 +176,32 @@ Note:
...
+#### What about SUSE in the community?
+
+@ul[](false)
+- OpenStack tests against openSUSE Leap using our packaging
+- When it breaks, they sometimes ping dirk on irc
+- And then... Sometimes... dirk is not available!
+- We need a better, more sustainable support for the community and take care of issues before they arise
+@ulend
+
+Help support OpenSUSE in OpenStack by:
```
Comment (dirkmueller) [inline 2019-01/PITCHME.md:193]: its still openSUSE

File: 2019-01/PITCHME.md [hunk 11]
```diff
@@ -176,5 +176,32 @@ Note:
...
+
+@ul[](false)
+- Helping provide stable support for OpenSUSE images by helping maintain diskimage-builder, supporting the mirrors for the OpenDev (nee OpenStack) Infra team
```
Comment (dirkmueller) [inline 2019-01/PITCHME.md:196]: openSUSE

File: 2019-01/PITCHME.md [hunk 12]
```diff
@@ -176,5 +176,32 @@ Note:
...
+- Help support community packages by contributing to the rpm-packaging project and keeping the Cloud:OpenStack:* OBS projects stable
```
Comment (dirkmueller) [inline 2019-01/PITCHME.md:197]: its not only about Cloud:OpenStack: . but about all of the distro. e.g. we released an unbound(1) security update that broke it completely (and that stopped the CI)

**Pull request merged**